server/app/services/applicant/UpdateServiceArea.java

```diff
@@ -0,0 +1,23 @@
+package services.applicant;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import services.Path;
+import services.geo.ServiceAreaInclusion;
+
+/** Represents the service area piece of the applicant's answers to an address question. */
+@AutoValue
+public abstract class UpdateServiceArea {
```

Comment:
**suggestion (blocking):** `Update` here sounds like a verb, recommend renaming to `ServiceAreaUpdate` so it's a noun-phrase

Reply:
👍 done